packages/app/app/models/order/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./order";
```

Comment:
Should we move this folder `models` outside the `app`? This looks something that can live in the same level as components

Reply:
Also this is shared between pages and components. So makes senso to live in the same level.

Reply:
fixed: https://github.com/SwaprHQ/stackly-ui/pull/23/commits/d950260c05cfac42e9a6372d7572331783ca866b